packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -632,28 +632,25 @@ class HotRunner extends ResidentRunner {
         unusedElements.addAll(await unusedReport);
 
       if (unusedElements.isNotEmpty) {
+        // TODO(devoncarew): Should we only show this once during a restart session?
+        // So, show it once, and don't show it again until they've restarted?
         unusedElementMessage =
-          '\nThe following program elements were changed by the reload, '
-          'but did not run when the view was reassembled. If this code '
-          'only runs at start-up, you will need to restart ("R") for '
-          'the changes to have an effect.';
+          'Some methods were changed by the reload but did not run when the view was reassembled;\n'
```

Comment:
"methods": The elements can be methods, field initializers or (when the VM has less data) whole classes. It's possible for the Flutter tool to tell which without any VM changes.

Reply:
I think we can be less precise here in the verbiage - something like `2 methods, 1 field, and 1 class were changed...` doesn't add a lot of value. Perhaps changing this to `items` instead (and possibly, having a bit more detail in the list of items about what each one is).

Good to know that the info is available from the protocol.

Reply:
I tried `items` here but it didn't look right w/ the rest of the message. I moved this back to `program elements`.